src/lib/renderPaperCheckoutLink .ts

```diff
@@ -51,19 +52,26 @@ export async function renderPaperCheckoutLink({
           case 'paymentSuccess': {
             const transactionId = e.data.id;
             PaperCheckoutResult.transactionId = transactionId;
+            PaperCheckoutResult.status = TransactionStatus.PAYMENT_SUCCEEDED;
             // TODO: Maybe we can resolve early if the user's want
             break;
           }
           case 'claimSuccessful': {
             const { id: transactionId, claimedTokens } = e.data;
             PaperCheckoutResult.transactionId = transactionId;
             PaperCheckoutResult.claimedTokens = claimedTokens;
+            PaperCheckoutResult.status = TransactionStatus.TRANSFER_SUCCEEDED;
             // let the user see the completed NFT for some short time before closing it
             await sleepForSeconds(3.5);
             resolve(result);
             drawer.close();
             break;
           }
+          case 'redirectAfterSuccess': {
+            const redirectUrl = e.data.redirectUrl;
+            window.location.replace(redirectUrl);
```

Comment:
Should we be pushing or replacing browser history? imo a push makes more sense here